lib/vector-core/src/schema/definition.rs

```diff
@@ -133,6 +152,86 @@ impl Definition {
         &self.log_namespaces
     }
 
+    /// Adds the `source_type` and `ingest_timestamp` metadata fields, which are added to every Vector source.
+    #[must_use]
+    pub fn with_standard_vector_source_metadata(self) -> Self {
```

Comment:
Is there any reason we're not setting the `meaning` here yet?

Reply:
This data isn't likely to be used by sinks, and I'm not sure we have really standardized what the "meanings" are yet. We are likely to revisit this in the future and I didn't want to pick something that might need to change. In the future if we do pick something it's just these 2 lines that need to change, rather than every source.